po/pt.po

```diff
@@ -182,11 +182,11 @@ msgstr "Diffs a mostrar?"
 
 #: clean.go:89
 msgid "Do you want to remove ALL AUR packages from cache?"
-msgstr "Quer remover todos os pacotes AUR da cache?"
+msgstr "Quer remover todos os pacotes AUR do cache?"
```

Comment:
I follow the priberam dictionary on this so cache should be feminine

https://dicionario.priberam.org/cache

Reply:
Yes Yes. If it is the case to follow the dictionary, cache is a feminine word, you are right.

Is it that, generally speaking, we tend to bring the "cache" to the male? Now why is a good question. :joy:

I will make the correction, thank you.